### PR TITLE
Simple finder chart

### PR DESCRIPTION
**Pull request by bmorris3**
This is an implementation of a simple finder chart, first implemented with examples [here](https://github.com/bmorris3/gsoc2015/blob/master/finder_chart.ipynb) (see also comments [here](https://github.com/bmorris3/gsoc2015/commit/88e06aebb7ee3cd71f4b9406302f7d62fc5266db)).

@jberlanga / @eteq / @cdeil Should we make some basic tests for this method? I wasn't following the `astroplan.plots` testing strategy very closely.

### CODE REVIEW

**Comment by cdeil** [comment]
@bmorris3 – Concerning tests for images, this should be done later (see #107), once someone figures out how to pin the plotting style across MPL versions (see https://github.com/astrofrog/pytest-mpl/issues/12).

For now, my suggestion would be that you do excercise the code you add here by adding a test and return the fig, as done in the one plot test example we have in astroplan. Then once we get pytest-mpl working smoothly, we add the decorator which does the image comparison and add the baseline image.

OK?

File: astroplan/plots/finder.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        ax.axhline(y=0.5*pixel_width, xmin=0.5-inner, xmax=0.5-outer,
+                   **reticle_style_kwargs)
+
+    # Labels, title, grid
+    ax.set(xlabel='RA', ylabel='DEC')
+    if target_name is not None:
+        ax.set_title(target_name)
+    ax.grid(grid)
+
+    # Redraw the figure for interactive sessions.
```
Comment (cdeil) [inline astroplan/plots/finder.py]: Is it a good idea to call `ax.figure.canvas.draw()` in plotting functions to trigger a redraw for Figures in interactive sessions?
Does this imply inefficient plotting (i.e. double-draw) somehow?

Is this relevant?
http://matplotlib.org/devdocs/users/whats_new.html#interactive-oo-usage

@bmorris3 – If no-one watching this issue comes up with a good answer within ~ a day you could ping e.g. Michael Droettboom with this question.

Comment (bmorris3) [inline astroplan/plots/finder.py]: This is how @jberlanga / @eteq / @adrn have the other plotting functions set up -- maybe they could illuminate the motivation for the redraw call?

Comment (jberlanga) [inline astroplan/plots/finder.py]: @bmorris3 - I don't recall the exact motivation for this--something to do
with multiple plots on top of one `ax` object not necessarily showing up in
an interactive session without a redraw?  @eteq?

Comment (bmorris3) [inline astroplan/plots/finder.py]: @cdeil Would you mind if I opened an issue on the `ax.figure.canvas.draw()` question, and merge this commit, which follows the form of the other plotting functions?

Comment (cdeil) [inline astroplan/plots/finder.py]: :+1:

**Comment by bmorris3** [comment]
How did we let me get away with not adding docs for this function? :x:

**Comment by adrn** [comment]
Oops... just looking at this now...

**Pull request merged**